examples/digest.js

```diff
@@ -1,8 +1,5 @@
-import { sleep } from "k6";
 import { crypto } from "k6/x/webcrypto";
 
-console.log(JSON.stringify(crypto));
-
 export default function () {
   const d = crypto.subtle.digest("SHA-256", "Hello, world!").then((hash) => {
```

Comment:
Indeed, this is a limitation at the moment. The specification relies on the fact that users will feed their ArrayBuffers themselves, using other specified constructs such as `TextEncoder`. However, I'd like to leave encoding of strings out of scope in this first iteration for mainly two reasons:
- Implementing the specification involves enough complexity, and I'd prefer not to add immediate deviation from it in this specific PR.
- It is possible to build small functions in pure Javascript doing the string encoding into array buffers without resorting to `TextEncoder`: https://github.com/grafana/xk6-webcrypto/pull/24/files#diff-4f6907dc6ec2ae10dc3b26dae0ff0135703a771ffb69fc756505b7704f836d76R40

I would open an issue to keep that topic on the roadmap, though, and I would like to address it in some way indeed separately. Let me know if that makes sense for you 🙇🏻

Reply:
I've updated the example to showcase it 👍🏻

Reply:
Thanks, but wouldn't adding support for string be a trivial change? Forcing users to use a helper encoding for such a common scenario doesn't seem friendly. That's assuming they can come up with it on their own, or find it in our examples or documentation.

I wouldn't call it a deviation from the spec, just augmenting it to support a very common usage scenario.

So I would prefer if we add it, though it can be done in a separate PR.

Reply:
I'm definitely not opposed to that, but I would indeed do it in a separate PR to avoid adding scope and complexity to this one 👍🏻

Reply:
tracked in #29